composer.json

```diff
@@ -11,34 +11,33 @@
             "homepage": "https://github.com/symfony-cmf/RoutingBundle/contributors"
         }
     ],
-    "minimum-stability": "dev",
-    "prefer-stable": false,
     "require": {
         "php": ">=5.3.9",
-        "symfony-cmf/routing": "~1.2",
-        "symfony/framework-bundle": "~2.3"
+        "symfony-cmf/routing": "^1.2",
+        "symfony/framework-bundle": "^2.3"
     },
     "require-dev": {
-        "symfony-cmf/core-bundle": "~1.1",
-        "symfony-cmf/testing": "~1.2,>=1.2.7",
-        "doctrine/phpcr-odm": "~1.3",
+        "symfony-cmf/core-bundle": "^1.1",
+        "symfony-cmf/testing": "dev-fix-travis-shell",
+        "doctrine/phpcr-odm": "^1.3@rc",
         "matthiasnoback/symfony-dependency-injection-test": "~0.6",
         "matthiasnoback/symfony-config-test": "0.*",
         "phpunit/php-code-coverage": "@stable",
-        "sonata-project/doctrine-phpcr-admin-bundle": "~1.1",
-        "symfony/monolog-bundle": "~2.3",
-        "doctrine/orm": "~2.3",
-        "doctrine/data-fixtures": "1.*,>=1.0.0-alpha3"
+        "sonata-project/doctrine-phpcr-admin-bundle": "^1.1",
+        "symfony/monolog-bundle": "^2.3",
+        "doctrine/orm": "^2.3",
+        "doctrine/doctrine-bundle": "@dev",
```

Comment:
The latest stable version has a big bug when creating a SQLite database. This is fixed, but a new version has not yet been released.